cmd/ocm/create/cluster/cmd.go

```diff
@@ -511,8 +525,19 @@ func run(cmd *cobra.Command, argv []string) error {
 	return nil
 }
 
-func shouldPromptForBYOVPC(areSubnetsProvided bool) bool {
-	return !args.byovpc.Enabled && args.ccs.Enabled && !areSubnetsProvided && args.interactive
+func shouldPromptForExistingVPC(areSubnetsProvided bool) bool {
+	return !args.existingVPC.Enabled && args.ccs.Enabled && !areSubnetsProvided && args.interactive
+}
+
+func wasClusterWideProxyReceived() bool {
+	return (args.clusterWideProxy.HTTPProxy != nil && *args.clusterWideProxy.HTTPProxy != "") ||
+		(args.clusterWideProxy.HTTPSProxy != nil && *args.clusterWideProxy.HTTPSProxy != "") ||
+		(args.clusterWideProxy.AdditionalTrustBundle != nil && *args.clusterWideProxy.AdditionalTrustBundle != "")
+}
+
+func isProxyEmpty() bool {
+	return *args.clusterWideProxy.HTTPProxy == "" && *args.clusterWideProxy.HTTPSProxy == "" &&
```

Comment:
Should there be a nil check on these before we dereference them?

Reply:
on the other hand, I think this should be left as is. As I said, there is no way they are nil. If they are, something is wrong.